tests/flows/erc20_source_erc20_destination.go

```diff
@@ -163,3 +163,158 @@ func ERC20SourceERC20Destination(network interfaces.Network) {
 	Expect(err).Should(BeNil())
 	Expect(balance).Should(Equal(bridgedAmount))
 }
+
+/**
+ * Deploy an ERC20 token source on the primary network
+ * Defers deploying ERC20Destination to Subnet A
+ * Bridges C-Chain example ERC20 tokens to Subnet A
+ * Confirms failure to execute message, since destination hasn't been deployed.
+ * Deploys ERC20Destination to Subnet A
+ * Invokes a retry of message execution
+ * Confirms message execution
+ */
+func ERC20DestinationNotYetDeployed(network interfaces.Network) {
```

Comment:
we've kept each test case to a separate file. `erc20_source_erc20_destination` just means a basic transfer, could possibly rename that. Can you move this test case to it's own file? `destination_bridge_not_deployed` or similar

Reply:
addressed in https://github.com/ava-labs/teleporter-token-bridge/pull/85/commits/eb2c309b9e6ceadab14b81ca5345a037a0f47149